test/parallel/test-repl.js

```diff
@@ -805,6 +805,16 @@ const tcpTests = [
   {
     send: `require(${JSON.stringify(moduleFilename)}).number`,
     expect: '42'
+  },
+  {
+    send: 'import comeOn from \'fhqwhgads\'',
```

Comment:
```suggestion
    send: 'import everybody from \'the limit\'',
```

Reply:
On second thought...

```suggestion
    send: 'import theLimit from \'everybody\'',
```

Reply:
Long descriptive names are preferable.

```suggestion
    send: 'import comeOn from \'fhqwhgadshgnsdhjsdbkhsdabkfabkveybvf\'',
```

Reply:
LOLOLOL

guess this will need to come in a followup